run_xnat2bids.py

```diff
@@ -565,13 +565,24 @@ async def launch_bids_validator(arg_dict, user, password, bids_root, job_deps):
     # Close connection
     connection.close()
     
-    # Define bids-validator singularity image path
-    simg=fetch_latest_simg("validator")
+    # Call latest bids-validator from xnat-tools via deno
+    simg=fetch_latest_simg('xnat-tools')
 
     for bids_experiment_dir in bids_experiments:
 
         # Build shell script for sbatch
-        sbatch_bids_val_script = f'""apptainer exec --no-home -B {bids_experiment_dir} {simg} bids-validator {bids_experiment_dir}""'
+        cmd = [
```

Comment:
I think we should abstract this part so and make sure we can re-use to call validator even without the conversion